client/app/src/Validator/Constraints/CommonPasswordValidator.php

```diff
@@ -2,29 +2,28 @@
 
 namespace App\Validator\Constraints;
 
-use RuntimeException;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\ConstraintValidator;
 
 class CommonPasswordValidator extends ConstraintValidator
 {
-    const TMP_ROOT_PATH = '/tmp/';
-    const PWNED_PW_URL = 'https://www.ncsc.gov.uk/static-assets/documents/PwnedPasswordsTop100k.txt';
+    public const TMP_ROOT_PATH = '/tmp/';
+    public const PWNED_PW_URL = 'https://www.ncsc.gov.uk/static-assets/documents/PwnedPasswordsTop100k.txt';
 
     private string $filePathCommonPasswords;
 
     private string $pwnedPasswordsUrl;
 
-    public function __construct()
-    {
-        $this->filePathCommonPasswords = self::TMP_ROOT_PATH.'commonpasswords.txt';
-        $this->pwnedPasswordsUrl = self::PWNED_PW_URL;
+    public function __construct(
```

Comment:
By passing these parameters to the constructor, we can use a fixtures file located in the TestData dir to replicate the common passwords file in the live environment.

Reply:
Just as an aside and not to be done in this ticket but when we did the risky passwords stuff there wasn't a symfony version available so this just reminded me. Now we can use this which I think does largely the same thing :): https://symfony.com/blog/new-in-symfony-4-3-compromised-password-validator

Reply:
> Just as an aside and not to be done in this ticket but when we did the risky passwords stuff there wasn't a symfony version available so this just reminded me. Now we can use this which I think does largely the same thing :): https://symfony.com/blog/new-in-symfony-4-3-compromised-password-validator

Sounds like a good idea.